repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py

```diff
@@ -412,7 +412,7 @@ def filter_out_out_pkgs(event_in_pkgs, event_out_pkgs):
     'RHEL 8 repositories that are intentionally excluded from the '
     'list of repositories used during the upgrade. '
     'See the report message titled "Excluded RHEL 8 repositories" '
-    'for details.\n The list of these packages (repositories: {}):'
+    'for details.\n The list of these packages:'
```

Comment:
```suggestion
    'for details.\nThe list of these packages:'
```
No need for the whitespace. It looks kinda strange in the report:
```
Summary: 2 packages will be skipped because they are available only in RHEL 8 repositories that are intentionally excluded from the list of repositories used during the upgrade. See the report message titled "Excluded RHEL 8 repositories" for details.
 The list of these packages:
- pkg1...
```